velox/functions/prestosql/types/CMakeLists.txt

```diff
@@ -22,6 +23,7 @@ velox_add_library(
 
 velox_link_libraries(
   velox_presto_types
+  GEOS::geos
```

Comment:
This should be added in a gated, separate invocation to make it optional as discussed in 12243